pom.xml

```diff
@@ -53,6 +53,8 @@
 
     <properties>
         <dropwizard.url>http://www.dropwizard.io/${project.version}</dropwizard.url>
+        <dropwizard.check.fail-owasp>true</dropwizard.check.fail-owasp>
+        <dropwizard.check.skip-owasp>false</dropwizard.check.skip-owasp>
```

Comment:
No need to introduce another property here. The OWASP dependency check plugin provides the `dependency-check.skip` property out of the box with the same default.